streamer/src/quic.rs

```diff
@@ -674,12 +679,22 @@ pub fn spawn_server_multi(
 mod test {
     use {
         super::*,
-        crate::nonblocking::{quic::test::*, testing_utilities::check_multiple_streams},
+        crate::nonblocking::{
+            quic::test::*,
+            testing_utilities::{check_multiple_streams, DEFAULT_NUM_SERVER_THREADS_FOR_TEST},
+        },
         crossbeam_channel::unbounded,
         solana_net_utils::bind_to_localhost,
         std::net::SocketAddr,
     };
 
+    fn rt_for_test() -> Runtime {
```

Comment:
if runtime provided as argument (RuntimeHandle passed to the streamer), this would allow to use tokio tests without creating runtime manually

Reply:
Yeah, something like that could be good. That seems to potentially fall in line with the "move runtime creation to higher level" PR.

I still think my change is an improvement given that I've consolidate 5 instances of duplicate code into this helper; it being consolidated should help ease any further refactoring as well